lsr_fingerprint.py

```diff
@@ -44,12 +47,40 @@
                         ]
                         for tmpl in tmpls:
                             tmplpath = join(dirpath, tmpl)
-                            with open(tmplpath) as fp:
-                                lines = fp.read()
-                            newlines = lines.replace(
-                                "{0}:{1}".format(role["reponame"], role["rolename"]),
-                                "system_role:{0}".format(role["lsrrolename"]),
-                            )
-                            if lines != newlines:
-                                with open(tmplpath, "w") as fp:
-                                    fp.write(newlines)
+                            with open(tmplpath) as ifp:
+                                lines = ifp.read()
+                            if lines.find(oldpair):
```

Comment:
`find` returns `-1` if the string was not found - https://docs.python.org/3/library/stdtypes.html#str.find